utils/sl/components/__init__.py

```diff
@@ -300,11 +300,14 @@ def define_components(reqs):
     retriever = GitRepoRetriever("https://github.com/spdk/spdk.git", True)
     reqs.define('spdk',
                 retriever=retriever,
-                commands=['./configure --prefix="$SPDK_PREFIX"' \
-                          ' --disable-tests --without-vhost --without-crypto' \
-                          ' --without-pmdk --without-vpp --without-rbd' \
-                          ' --with-rdma --with-shared' \
-                          ' --without-iscsi-initiator --without-isal' \
+                commands=['cd dpdk; '                            \
```

Comment:
joining the commands with "&&" might be preferable to ensure we don't continue if one of the commands fails

Reply:
If one of the `cd dpdk` or `git fetch` fails, the `git checkout` will also fail, producing a failure for that `commands` list item.

Looking at https://github.com/daos-stack/daos/blob/bcbce11f4ed838835b074083e31342914589ca9f/utils/sl/prereq_tools/base.py#L284-L294 it would seem that the failed `git checkout` would abort the build at that point, making the error quite localized.

Do you think the above is sufficient at this time @tom or do you see additional value in doing the `s/;/&&/g` in that `commands` item?

Alternatively, I think a better option than `s/;/&&/g` in that list of commands would be to (eventually) apply this patch:

```patch
diff --git a/utils/sl/prereq_tools/base.py b/utils/sl/prereq_tools/base.py
index 911a93f4d2..c6b5dc4892 100644
--- a/utils/sl/prereq_tools/base.py
+++ b/utils/sl/prereq_tools/base.py
@@ -288,7 +288,7 @@ class Runner():
                 retval = True
             else:
                 print('RUN: %s' % command)
-                if subprocess.call(command, shell=True,   # nosec
+                if subprocess.call(['set -e; '] + command, shell=True, # nosec
                                    env=self.env['ENV']) != 0:
                     retval = False
                     break
```

Then `;` or `&&` can be used equally and interchangeably in the future.

Reply:
Fine for now, the latter fix would be good when you can.